express-js/controllers/product.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const rootDir = require('../utils/path');
+const Product = require('../models/product');
+
+exports.getAddProduct = (req, res, next) => {
+    console.log("first request");
+    res.sendFile(path.join(rootDir, 'views', 'add-product.html'));
+}
+
+exports.addProduct = (req, res, next) => {
+    const product = new Product(req.body.title);
+    product.save();
+    res.redirect('/');
+}
+
+exports.getProduct = (req, res, next) => {
+    const product = Product.fetchAll();
+    console.log(product);
```

Comment:
Please remove console logs.
Never log api responses as it may have confidential info

Reply:
okay, I will remove that.